### ISSUE TITLE
Re-open to last position after quickunlock

### ISSUE THREAD

**Issue by tunbridgep** [PhilippC/keepass2android#1931]
Current behavior:
- unlock database
- go into a group
- wait for timeout and automatic lock or lock manually
- unlock via quickunlock
- you're at the database root

What I expected:
- it would open up in the group I was in before locking

This is especially annoying with a big database (10,000+ entries) with lots of groups and subgroups.

Obviously this would require storing a database path, plus some people might prefer how it currently is, so this should be opt in.

**Comment by chris47368** [comment]
Yeah that bugs me a bit too, maybe a checkbox within the settings to control if the database remains in its last location after quick unlock or goes back to root(current behaviour) after quick unlock would be nice.

To further this: I suggest a "go back home/root" button in the top corner of the app would be nice too, just as there is a 'back' button in the left top corner of the app as current.

**Comment by cjbarth** [comment]
Those interested in this might want to vote for https://github.com/PhilippC/keepass2android/issues/22
